### ISSUE TITLE
Deleting app from serverpilot does not delete relevant app ssl.config

### ISSUE THREAD

**Issue by ilearnbydoing** [lesaff/serverpilot-letsencrypt#62]
Hi, 
Deleting the app from serverpilot dashboard does not delete it's .ssl.conf file which at the time of renewal causes issue of falling of nginx-sp service.
I had to manually run "letsenrypt renew" to find the culprit .ssl.conf file and then restart nginx-sp, 
Is there any way to delete specific ssl.conf while deleting its corresponding app form serverpilot web panel?

**Comment by caroseuk** [comment]
This is correct, as the script is not a part of Server Pilot so any actions performed through the Server Pilot control panel won't take affect.

The script could probably be tweaked so that when the cron job runs, it looks for any orphaned .ssl.conf files and removes them if there are no corresponding apps on the server. Will have a look and see what is possible :)
